azure/azure-external-release-pipeline.yml

```diff
@@ -9,6 +9,9 @@ resources:
       type: github
       name: NHSDigital/api-management-utils
       endpoint: NHSDigital
+  pipelines:
+    - pipeline: build_pipeline
+      source: "Electronic-Prescription-Service-Build"
 
```

Comment:
Does this need the trigger filter to be consistent with internal release?

```
trigger:
      branches:
        include:
          - refs/tags/v*
```

So only tagged releases from master trigger?